pyccel/codegen/utilities.py

```diff
@@ -42,40 +42,64 @@
 #==============================================================================
 
 def not_a_copy(src_folder, dst_folder, filename):
-    """ Check if the file filename present in src_folder
-    is a copy of the file filename present in dst_folder
-    or if the source file has been updated since the last
-    copy
+    """
+    Check if the file is different between the source and destination folders.
+
+    Check if the file filename present in src_folder is different (not a copy)
+    from the file filename present in dst_folder. This is done by checking if
+    the source file has been modified more recently than the destination file.
+    This would imply that it has been modified since the last copy.
+
+    Parameters
+    ----------
+    src_folder : str
+        The folder where the file was defined.
+
+    dst_folder : str
+        The folder where the file is being used.
+
+    filename : str
+        The name of the file.
+
+    Returns
+    -------
+    bool
+        False if the file in the destination folder is a copy of the file in the
+        source folder, True otherwise.
     """
     abs_src_file = os.path.join(src_folder, filename)
     abs_dst_file = os.path.join(dst_folder, filename)
-    src_mod_time = os.path.getatime(abs_src_file)
-    dst_mod_time = os.path.getatime(abs_dst_file)
+    src_mod_time = os.path.getmtime(abs_src_file)
+    dst_mod_time = os.path.getmtime(abs_dst_file)
     return src_mod_time > dst_mod_time
 
 #==============================================================================
 def copy_internal_library(lib_folder, pyccel_dirpath, extra_files = None):
     """
+    Copy an internal library to the specified pyccel directory.
+
     Copy an internal library from its specified stdlib folder to the pyccel
     directory. The copy is only done if the files are not already present or
     if the files have changed since they were last copied. Extra files can be
     added to the folder if and when the copy occurs (e.g. for specifying
-    the numpy version compatibility)
+    the numpy version compatibility).
```

Comment:
In theory we should capitalize Pyccel and Numpy...

Reply:
Fixed 45e7c0b9494cc4ca8d8b6a98c5c93c5a70d9cefd